scripts/build/companion_libs/050-zlib.sh

```diff
@@ -94,20 +94,20 @@ do_zlib_backend() {
         "${extra_config[@]}"
 
     CT_DoLog EXTRA "Building zlib"
-    CT_DoExecLog ALL make ${JOBSFLAGS}
+    CT_DoExecLog ALL ${make} ${JOBSFLAGS}
```

Comment:
I am not sure why this is needed. The first thing crosstool-ng does (before executing any scripts in `scripts/build`) is setting up a tools override directory (by default, `.build/tools/bin`) where shell wrappers point to the host tools determined by configure. This directory is then added to the `$PATH` before first.